receipt_upload/receipt_upload/merchant_resolution/resolver.py

```
@@ -115,12 +115,8 @@ def merchant_name_matches_receipt(
         return True  # No receipt text to validate against
 
     # Build a token set from the top N receipt lines (by y-coordinate)
-    sorted_lines = sorted(
-        lines, key=lambda l: l.calculate_centroid()[1]
-    )
-    receipt_text = " ".join(
-        l.text for l in sorted_lines[:n_lines] if l.text
-    )
+    sorted_lines = sorted(lines, key=lambda l: l.calculate_centroid()[1])
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)